src/tasks/WorkloadBuildTasks/WorkloadBuildTasks.csproj

```diff
@@ -11,9 +10,4 @@
 
     <PackageReference Include="Microsoft.Build.Tasks.Core" Version="$(MicrosoftBuildTasksCoreVersion)" />
   </ItemGroup>
-
-  <Target Name="PublishBuilder"
-          AfterTargets="Build"
-          DependsOnTargets="Publish" />
-
```

Comment:
Do you remember why this existed?

Reply:
No, but it seems like a mistake. So, I'm waiting to see how the CI run goes.

Reply:
Maybe also double check the nuget package content. That should be a good indicator to see if something was previously added from the publish folder.

Reply:
This is used only for preparing a sdk+workload installation for testing only.